### PR TITLE
[HOPSWORKS-1586] Add Datasets and project members pages

### PR DESCRIPTION
**Pull request by tkakantousis**

### CODE REVIEW

File: docs/user_guide/hopsworks/dataSetShare.rst [hunk 1]
```diff
@@ -1,21 +1,84 @@
...
+    :scale: 30%
     :align: center
     :figclass: align-center
 
-    Share dataset
+Unshare
+-------
+
+Data Sets can be ``unshared`` by `data owners`, either from the parent project or from the project they have been
+shared with. To do that, users of the parent project can right-click on the Data Set, click ``UnShare from`` and then
```
Comment (SirOibaf) [inline docs/user_guide/hopsworks/dataSetShare.rst]: `unshared` or `UnShare`?

File: docs/user_guide/hopsworks/dataSetShare.rst [hunk 2]
```diff
@@ -1,21 +1,84 @@
...
+.. _data-set-access-control:
+Data Set access-control
+-----------------------
+
+Inviting other users to a project and sharing a Data Set are ways to allow members of an organization to provide
+fine-grained access control to original data to other members/departments without having to relinquish full control
+over the data. Essentially that means that data organized in Data Sets can be shared with other projects so that
+other users of Hopsworks can access and make use of this data in their AI pipelines and programs.
+
+By default, Data Scientints of the project owning the Dataset and members of projects which a Data Set has been shared
```
Comment (SirOibaf) [inline docs/user_guide/hopsworks/dataSetShare.rst]: Dataset and Data Set in the same sentence

File: docs/user_guide/hopsworks/dataSetShare.rst [hunk 3]
```diff
@@ -1,21 +1,84 @@
...
+with are allowed only ``read-only`` access to all file and directories of the Data Set. There are two more access
+levels that give stronger privileges to projects that can access this Data Set. All access-levels are:
```
Comment (SirOibaf) [inline docs/user_guide/hopsworks/dataSetShare.rst]: I think it's higher privileges

File: docs/user_guide/hopsworks/dataSetShare.rst [hunk 4]
```diff
@@ -1,21 +1,84 @@
...
+
+- Owner only: Sets default permissions setting of a Dataset. Only Data Owners will be able to upload/remove files, either via the Dataset Browser or via Jobs and Jupyter notebooks.
```
Comment (SirOibaf) [inline docs/user_guide/hopsworks/dataSetShare.rst]: Same Dataset - Data Sets inconsistency

Comment (SirOibaf) [reply docs/user_guide/hopsworks/dataSetShare.rst]: Actually this is not true. Owner only maps to 750 - which means that only the data owner which created the dataset can write there. Maybe in Hopsworks we upload as superuser and then chown - but you won't be able to write there from Jobs and Jupyter.

File: docs/user_guide/hopsworks/dataSetShare.rst (hunk 1 above)
Comment (tkakantousis) [reply docs/user_guide/hopsworks/dataSetShare.rst]: I'm copying whatever is in the UI , which is ``UnShare from``. In another jira we can fix the UI

File: docs/user_guide/hopsworks/dataSetShare.rst (hunk 2 above)
Comment (tkakantousis) [reply docs/user_guide/hopsworks/dataSetShare.rst]: Fixed. For reference, this inconsistency is spread out in many repos and it was decided at some point to refactor everything to Dataset. Another JIRA's work.

File: docs/user_guide/hopsworks/dataSetShare.rst (hunk 4 above)
Comment (tkakantousis) [reply docs/user_guide/hopsworks/dataSetShare.rst]: The sad thing is I copied this from the Hopsworks UI modal.

Comment (tkakantousis) [reply docs/user_guide/hopsworks/dataSetShare.rst]: Yes in Hopsworks we do that, probably we shouldn't

**Pull request merged**